src/patternfly/demos/Toolbar/examples/toolbar-attribute-value-checkbox-select-filter-example.hbs

```diff
@@ -0,0 +1,107 @@
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-checkbox-select-filter-example"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-reveal-on-lg" data-toolbar-group--IsExpanded="false"}}
+      {{> data-toolbar-toggle}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-filter-group"}}
+      {{#> data-toolbar-item}}
+        {{#> select id="select-two" select-toggle--icon="fas fa-filter"}}
```

Comment:
Can you `concat()` the value of `data-toolbar--id` with this `id` (and the ones in the other examples), so we can copy/paste these demos and the only unique ID we have to use is `data-toolbar--id`?